src/OpenTelemetry.Exporter.Jaeger/AssemblyInfo.cs

```diff
@@ -24,5 +24,5 @@
 #else
 [assembly: InternalsVisibleTo("OpenTelemetry.Exporter.Jaeger.Tests")]
 [assembly: InternalsVisibleTo("Benchmarks")]
-[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2] // Used by Moq.
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")] // Used by Moq.
```

Comment:
Do you know why do we need this?

Reply:
The "DynamicProxyGenAssembly2" thing? Or the non-signed build?

* The "DynamicProxyGenAssembly2" thing I'm assuming it's so the Moq library can access the internals we ask it to mock in the unit tests.
* Non-signed builds, no idea.